src/collections/components/CollectionSearchForm/CollectionSearchForm.validation.tsx

```diff
@@ -4,10 +4,12 @@ import { CollectionStatus } from '../../../api/generatedTypes';
 export const validationSchema = yup.object({
   title: yup.string().min(3),
   author: yup.string().min(3),
+  labels: yup.array(),
   status: yup.mixed<CollectionStatus>().oneOf(Object.values(CollectionStatus)),
```

Comment:
This is as far as I went with validation - just an array. https://github.com/Pocket/curation-admin-tools/pull/968 will introduce validation.